lib/bitmap-file-helper.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const fs = require('fs');
+const Bitmap = require('../model/bitmap-constructor.js').Bitmap;
+
+module.exports = exports = {};
+
+exports.load = function(filepath, callback) {
+  fs.readFile(filepath, function (err, data) {
+    if(err) return callback(err);
+    var bm = new Bitmap(data);
+    //TODO: Possibly store filepath in the bm object, or pass as
+    //      constructor param.
+    callback(null, bm);
```

Comment:
@dbecker4130 @geoffsimons @meganreardon great use of the `(err, result)` callback pattern along with passing a `null` value as the first parameter